modules/risk-deployment-details-security-context.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * operating/evaluate-security-risks.adoc
+:_module-type: CONCEPT
+[id="risk-deployment-details-security-context_{context}"]
+= Security context section
+
+The security context section shows details about the following:
```

Comment:
I mention this elsewhere in the PR but perhaps "Security Context" should be capitalized in the header and body text if it is spelled that way in a displayed name in a tab or section?